django/db/backends/postgresql/creation.py

```diff
@@ -28,8 +29,16 @@ def sql_table_creation_suffix(self):
             template=test_settings.get('TEMPLATE'),
         )
 
+    def _database_exists(self, cursor, database_name):
+        cursor.execute('SELECT 1 FROM pg_database WHERE datname = %s', [strip_quotes(database_name)])
```

Comment:
I wonder what the purpose of `strip_quotes()` is -- usually that handles table names on Oracle.

Reply:
Database name is already quoted at this point (see [base/creation.py#L164](https://github.com/django/django/blob/master/django/db/backends/base/creation.py#L164) and [postgresql/creation.py#L52](https://github.com/django/django/blob/master/django/db/backends/postgresql/creation.py#L52)), that's why I added `strip_quotes()`. Alternatively we can pass an unquoted database name to the `_execute_create_test_db()`, but it will be redundant, IMO.